tests/unit/test_plots.py

```diff
@@ -91,4 +119,15 @@ def test_optim_plots(self, optim):
         pybop.plot_parameters(optim)
 
         # Plot the cost landscape with optimisation path
-        pybop.plot_cost2d(optim.cost, optim=optim, steps=5)
+        pybop.plot2d(optim, steps=5)
+
+    @pytest.mark.unit
+    def test_with_ipykernel(self, dataset, cost, optim):
+        import ipykernel
+
+        print(ipykernel.__version__)
```

Comment:
Perhaps an assert instead of a print will ensure this doesn't get missed? 
```suggestion
        assert ipykernel.__version__ >= 0.6
```